cmd/admin/unpause_account.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"strconv"
+
+	sapb "github.com/letsencrypt/boulder/sa/proto"
+	"golang.org/x/exp/maps"
+)
+
+// subcommandUnpauseAccount encapsulates the "admin unpause-account" command.
+type subcommandUnpauseAccount struct {
+	batchFile string
+	regID     int64
+}
+
+var _ subcommand = (*subcommandUnpauseAccount)(nil)
+
+func (u *subcommandUnpauseAccount) Desc() string {
+	return "Administratively unpause an account to allow certificate issuance attempts"
+}
+
+func (u *subcommandUnpauseAccount) Flags(flag *flag.FlagSet) {
```

Comment:
Totally optional suggestion: implement a parallelism flag, like some of the other subcommands have.

Reply:
I am going to skip this for now. This sounds like a good future improvement.